torch/csrc/api/include/torch/nn/functional/loss.h

```diff
@@ -589,6 +593,32 @@ inline Tensor cross_entropy(
       options.reduction());
 }
 
+// ============================================================================
+
+namespace detail {
+inline Tensor binary_cross_entropy_with_logits(
+  const Tensor& input, const Tensor& target,
+  const Tensor& weight, const Tensor& pos_weight,
+  BCEWithLogitsLossOptions::reduction_t reduction) {
```

Comment:
To follow the ordering in the Python version, maybe `reduction` before `pos_weight`?